src/core/tests/mutations.test.tsx

```diff
@@ -364,7 +386,7 @@ describe('mutations', () => {
     consoleMock.mockRestore()
   })
 
-  test('cancel mutation should not call mutationFn if the current retrier is undefined', async () => {
+  test('cancel mutation should not call mutationFn if the current retier is undefined', async () => {
```

Comment:
This looks like an accidental change?

```suggestion
  test('cancel mutation should not call mutationFn if the current retrier is undefined', async () => {
```

Reply:
Tried to spell, turns out I can't, I will revert it.